articles/integrations/aws/index.md

```diff
@@ -0,0 +1,11 @@
+---
+description: How to integrate AWS with Auth0 to setup SSO or obtain AWS tokens.
+---
+
+# Auth0-Amazon Web Services (AWS) Integrations
+
+Auth0 supports integration with AWS' Identity and Access Management (IAM) service, allowing you to:
+
+ * [Perform SSO with the AWS Dashboard](/aws/sso)
```

Comment:
The first two links are broken, you should either update them including the `/integrations` part of add url metadata at the docs.